src/index.js

```diff
@@ -0,0 +1,39 @@
+import { DateTime } from 'luxon';
+
+import Library from './modules/library.js';
+// eslint-disable-next-line max-classes-per-file
+
+// eslint-disable-next-line no-unused-vars
```

Comment:
- [ ] As pointed out by the previous reviewer, you are not supposed to disable any linter rule, the linters rules are there to ensure yo follow the best practices, I'll suggest you remove the disable rules and fix the issue correctly. :+1: :smile: